.tekton/main-pipeline.yaml

```diff
@@ -175,79 +160,162 @@ spec:
       operator: in
       values: [ "true" ]
     workspaces:
-    - name: output
-      workspace: workspace
     - name: basic-auth
       workspace: git-auth
 
   - name: determine-image-tag
     params:
-    - name: tag-suffix
+    - name: TAG_SUFFIX
       value: $(params.output-tag-suffix)
-    runAfter:
-    # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
-    - clone-repository
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
     taskRef:
-      name: determine-image-tag
-    workspaces:
-    - name: source
-      workspace: workspace
+      name: determine-image-tag-oci-ta
 
   - name: copy-subscription-manager-activation-key
-    runAfter:
-    - determine-image-tag
+    params:
+      - name: SOURCE_ARTIFACT
+        value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+      - name: ociStorage
+        value: $(params.output-image-repo):konflux-$(params.revision).subscription-manager
+      - name: ociArtifactExpiresAfter
+        value: $(params.oci-artifact-expires-after)
     workspaces:
-    - name: source
-      workspace: workspace
     - name: subscription-manager-activation-key
```

Comment:
Since this `subscription-manager-activation-key` and `fetch-external-networks` became lengthy, I'd ask you to kindly move them to external yalm files.

Reply:
Done in https://github.com/stackrox/stackrox/pull/11743/commits/246db196febe1c0b1e9c1442404b656e7a5538f6.